templates/slim/section.html.slim

```diff
@@ -1,6 +1,9 @@
 / hide slides on %conceal, %notitle and named "!"
 - titleless = (title = self.title) == '!'
 - hide_title = (titleless || (option? :notitle) || (option? :conceal))
+
+/ slide options processing
+- data_background = (attr? 'background') ? (image_uri(attr 'background')) : nil
```

Comment:
Agreed. This must definitely go in. I use background colors all the time.
How could I forget...

On Tue, Mar 29, 2016, 4:52 PM Dan Allen <notifications@github.com> wrote:

> In templates/slim/section.html.slim
> <https://github.com/asciidoctor/asciidoctor-reveal.js/pull/52#discussion_r57797873>
> :
>
> > @@ -1,6 +1,9 @@
> >  / hide slides on %conceal, %notitle and named "!"
> >  - titleless = (title = self.title) == '!'
> >  - hide_title = (titleless || (option? :notitle) || (option? :conceal))
> > +
> > +/ slide options processing
> > +- data_background = (attr? 'background') ? (image_uri(attr 'background')) : nil
>
> Great point. In my old patch, I had the following:
>
>   - if (bg = attr 'data-background') && bg =~ /\.(bmp|gif|jpe?g|png|svg)$/
>     - bg_image = image_uri bg
>     - bg = nil
>   - else
>     - bg_image = (attr? 'data-background-image') ? image_uri(attr 'data-background-image') : nil
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor-reveal.js/pull/52/files/e5bb7247eeab4a03bdab87c4705ac650f515b60b..d3534081ef6ce16c36251e98b309a4a38b5b31bb#r57797873>
>